libproto/src/lib.rs

```diff
@@ -296,6 +285,40 @@ macro_rules! impl_convert_for_struct {
     };
 }
 
+impl Into<MsgClass> for Vec<u8> {
+    fn into(self) -> MsgClass {
+        MsgClass::RawBytes(self)
+    }
+}
+
+impl Into<Message> for Vec<u8> {
+    fn into(self) -> Message {
+        Message::init(OperateType::BROADCAST, ZERO_ORIGIN, self.into())
+    }
+}
+
+macro_rules! impl_convert_for_struct_in_msg {
```

Comment:
Why we should use the same name for same thing in different places?
Look at this macro!
We can add some logic easier for the structs.
Better than writing some similar codes again and again.